### ISSUE TITLE
 Can we compute the accuracy in the training? 

### ISSUE THREAD

**Issue by LinJM** [igormq/ctc_tensorflow_example#4]
Hello, @igormq , I have a question, can we compute the accuracy in  training step for the each batch data. 
I use your project's fork  https://github.com/synckey/tensorflow_lstm_ctc_ocr , in this project , It want to process ocr problem. But I was confused with the accuracy compute, beacuse you can't compute the accuracy in every batch, in some batch, the output target's length will smaller than the label's length.
https://github.com/synckey/tensorflow_lstm_ctc_ocr/blob/master/lstm_and_ctc_ocr_train.py#L39

**Comment by igormq** [comment]
Hi @LinJM, yes you can calculate the accuracy for each batch data, but not using the default accuracy metric. For this kind of problem we use the [edit distance](https://en.wikipedia.org/wiki/Edit_distance), that [is being](https://github.com/synckey/tensorflow_lstm_ctc_ocr/blob/master/lstm_and_ctc_ocr_train.py#L75) calculating. :)

**Comment by igormq** [comment]
Closed #3

**Comment by chaine09** [comment]
@igormq I almost always get 100% accuracy for each batch data
